### PR TITLE
Add description of community proposal process.

### PR DESCRIPTION
**Pull request by keathley**
This is a brief description of the current process for submitting a feature proposal to the core team as discussed with @josevalim. I'm not sure I managed to match the voice of the document correctly so any input there would be great. And obviously if I'm misrepresenting the existing process please correct me.

### CODE REVIEW

**Comment by josevalim** [comment]
Thanks @keathley. I have slightly reordered the paragraphs to keep everything about the community and language development streamlined. Let me know what you think.

**Comment by josevalim** [comment]
Thanks @GregMefford! I believe our README already mentions this and other more mechanical steps. But if you find something is lacking, a PR there is welcome!

**Comment by keathley** [comment]
@josevalim The reordering seems good to me.

File: development.markdown [hunk 1]
```diff
@@ -14,15 +14,17 @@ Elixir runs on top of the Erlang Virtual Machine, which provides a scalable and
...
 
+The language development is open, both in terms of source code and of collaborations. All features and bug fixes planned for the next releases can be found [in the issues tracker](https://github.com/elixir-lang/elixir/issues). Features that may cause a larger impact on the ecosystem are first proposed to the community in [the Elixir mailing list](https://groups.google.com/group/elixir-lang-core) as well as in [the "Elixir News" section in the Elixir Forum](https://elixirforum.com/c/elixir-news). 
+
+Community members are welcome to propose new features for Elixir. Before submitting a proposal, members are encouraged to gather feedback from around the community in whatever venue seem best. However, in order for a proposal to be considered for inclusion by the Elixir Core team, it must go through the Elixir mailing list. This often includes discussion and refinement of the proposal. The Elixir Core team has the final say on whether a proposal is accepted or rejected. While members are encouraged to gain support from the rest of the community, popularity does not mean that a proposal will be accepted.
+
```
Comment (lancehalvorsen) [inline development.markdown]: I believe there's a typo here. I would suggest, " . . . whatever venues seem best."

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**